tests/ci-occm-e2e.sh

```diff
@@ -106,7 +106,8 @@ ansible-playbook -v \
   --ssh-common-args "-o StrictHostKeyChecking=no" \
   tests/playbooks/test-occm-e2e.yaml \
   -e github_pr=${PR_NUMBER} \
-  -e github_pr_branch=${PR_BRANCH}
+  -e github_pr_branch=${PR_BRANCH} \
+  -e octavia_provider=ovn
```

Comment:
Did you mean to hardcode it here?

Reply:
Yes, I wanted to test this without making changes to kubernetes/test-infra yet. Looks like it *almost* works. One test fails, but doesn't fail the job, so this is pretty much WiP.